e2e-tests/ukef/cypress.config.js

```diff
@@ -35,7 +35,7 @@ module.exports = defineConfig({
     openMode: 0,
   },
   e2e: {
-    baseUrl: 'http://localhost:5000',
+    baseUrl: 'http://localhost',
```

Comment:
Good spot -- I'm not sure the reason why this was `5000` initially -- as we do probably want this to hit our reverse proxy.